sample-apps/donet-sample-app/README.md

```diff
@@ -0,0 +1,36 @@
+# AWS Distro for OpenTelemetry .NET - Integration Testing App
+
+This application validates the continual integration with the AWS Distro for OpenTelemetry .NET and the AWS X-Ray back-end service. Validation is done using the [AWS Test Framework for OpenTelemetry](https://github.com/aws-observability/aws-otel-test-framework).
+
+## Application interface
+
+The application exposes the following routes:
+1. /
+    1. Ensures the application is running
+2. /outgoing-http-call
+    1. Makes a HTTP request to aws.amazon.com (http://aws.amazon.com/)
+3. /aws-sdk-call
+    1. Makes a call to AWS S3 to list buckets for the account corresponding to the provided AWS credentials
+4. /outgoing-sampleapp
+    1. Makes a call to all other sample app ports configured at `<host>:<port>/outgoing-sampleapp`. If none available, makes a HTTP request to www.amazon.com (http://www.amazon.com/) 
+
+## Running the integration testing application locally
+
+This application **lacks** dependencies for AWS X-Ray trace id generator, propagator and AWS client instrumentation and is intended for github workflow. If you want to run it locally, follow steps below:
```

Comment:
what dependency is lacking? I can see the dependency in [donet-sample-app.csproj](https://github.com/aws-observability/aws-otel-community/pull/187/files#diff-f82da6d9d8ab7e8c4ff78e93f1c150449492238d7dc984475c1aaadc6c409b8f)

Reply:
Updated the doc